flink-learning-common/src/test/java/com/zhisheng/common/utils/DateUtilTests.java

```diff
@@ -0,0 +1,60 @@
+package com.zhisheng.common.utils;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+import java.util.Date;
+
+import static com.zhisheng.common.utils.DateUtil.*;
+import static org.junit.Assert.*;
```

Comment:
We have had some feedback that wildcard imports are not desirable so probably better to import each assert type used or call as `Assert.assertEquals`, etc (unless this approach matches the rest of the project)

Reply:
Done